commands/core/help.js

```diff
@@ -37,12 +37,12 @@ module.exports = {
                         { name: 'Name', value: command.name, inline: true },
                         { name: 'Category', value: command.category, inline: true },
                         { name: 'Aliase(s)', value: command.aliases.length < 1 ? 'None' : command.aliases.join(', '), inline: true },
-                        { name: 'Utilisation', value: command.utilisation.replace('{prefix}', client.config.discord.prefix), inline: true },
+                        { name: 'Utilisation', value: `${client.config.discord.prefix} ${command.utilisation}`, inline: true },
```

Comment:
```suggestion
                        { name: 'Utilisation', value: `${client.config.discord.prefix}${command.utilisation}`, inline: true },
```
ommand prefix and name are not separated by a space, this should be removed.